src/main/java/com/mrbysco/unhealthydying/config/UnhealthyConfig.java

```diff
@@ -31,7 +31,7 @@
 
 			minimumHealth = builder
 					.comment("Minimum amount of health the player can end up with (2 = 1 heart) [default: 2]")
-					.defineInRange("minimumHealth", 2, 1, Integer.MAX_VALUE);
+					.defineInRange("minimumHealth", 6, 1, Integer.MAX_VALUE);
```

Comment:
Why did you change the default config value?

Reply:
Sorry, you may remove these changes